projects/common/src/lib/controls/api-access/api-access.component.ts

```diff
@@ -40,6 +40,8 @@ export class ApiAccessComponent implements OnChanges, OnInit {
   @Output('regenerated')
   public Regenerated: EventEmitter<string>;
 
+  protected toggleKeyVisibility: boolean;
```

Comment:
move up to the //Fields section

Reply:
We don't need this, just forgot to remove it.